contracts/CompoundGovernor.sol

```diff
@@ -33,6 +33,54 @@ contract CompoundGovernor is
     GovernorSettableFixedQuorumUpgradeable,
     OwnableUpgradeable
 {
+    /// @notice Emitted when the expiration of a whitelisted account is set or updated.
+    /// @param account The address of the account being whitelisted.
+    /// @param expiration The timestamp until which the account is whitelisted.
+    event WhitelistAccountExpirationSet(address account, uint256 expiration);
+
+    /// @notice Emitted when the whitelistGuardian is set or changed.
+    /// @param oldGuardian The address of the previous whitelistGuardian.
+    /// @param newGuardian The address of the new whitelistGuardian.
+    event WhitelistGuardianSet(address oldGuardian, address newGuardian);
+
+    /// @notice Emitted when the proposalGuardian is set
+    /// @notice Emitted when the proposal guardian is set or updated.
+    /// @param oldProposalGuardian The address of the previous proposal guardian.
+    /// @param oldProposalGuardianExpiry The expiration timestamp of the previous proposal guardian's role.
+    /// @param newProposalGuardian The address of the new proposal guardian.
+    /// @param newProposalGuardianExpiry The expiration timestamp of the new proposal guardian's role.
+    event ProposalGuardianSet(
+        address oldProposalGuardian,
+        uint96 oldProposalGuardianExpiry,
+        address newProposalGuardian,
+        uint96 newProposalGuardianExpiry
```

Comment:
The bravo governor uses `uint256` for `newProposalGuardianExpiry`, and I'm not really sure why? I've kept it `uint96` for consistency with the `oldProposalGuardianExpiry`

Reply:
I kept it as `uint96 newProposalGuardianExpiry`. Thought if it was a mistake to mark it as `uint256`, this is our chance to update it to use `uint96`.